articles/portal/ptl-ref-portal-api.md

```diff
@@ -212,593 +212,45 @@ Returns a response with a list of up to 10 VMs
 ```
 
 ## GET /api/my_calls
```

Comment:
I wonder if it'd be a better idea to link back to the Markdown file in this repository at a commit before this one gets merged in, instead of leaving the old docs live. I'm not sure on this one, but I'm also not sure why we're keeping references to these docs at all if LANDESK is going.

Reply:
> I'm (...) not sure why we're keeping references to these docs at all if LANDESK is going.

If an API user wants to know why their program has stopped working, this is somewhere they're likely to look. It shouldn't be kept forever.